README.md

```diff
@@ -1,24 +1,68 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Rails Budget App
 
-Things you may want to cover:
+> Rails budget app is a expense tracker app where an authorized user can log and add transaction to specific categories.
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on Rails
+- RSPEC-RAILS, Capybara
 
-* Configuration
+### Unfortunately, my LOOM is not working. I'm trying this for a couple of hours.
```

Comment:
Please fix this before merging ✊

Reply:
I'm trying for more than 2 hours but it's not working. I un-installed and re-installed ti again but still same.